lib/services/dapi/Dapi.js

```diff
@@ -76,7 +76,7 @@ class Dapi extends NodeJsService {
           starting = false;
         }
       } catch (error) {
-        if (error.message.indexOf("Merkle root from the diff doesn't match calculated merkle root after diff is applied") > -1) {
+        if (error.message.indexOf("DAPI RPC error: getBestBlockHeight: TypeError: Cannot read property 'length' of undefined") > -1 || error.message.indexOf("DAPI RPC error: getBestBlockHeight: Error: No MNs in list. Can't connect to the network.") > -1) {
```

Comment:
Shall we fix it in a separate PR?

Reply:
By the way, why do we change it? :)

Reply:
I don't know) @Cofresi @antouhou  do you have any ideas with this new behavior?

Reply:
Could we fix it separately? It's out of scope.

Reply:
without this fix dapi tests failed. or you suggest to disable them at all?